lib/x/response_parser.rb

```diff
@@ -39,9 +39,12 @@ class ResponseParser
     def parse(response:, array_class: nil, object_class: nil)
       raise error(response) unless response.is_a?(Net::HTTPSuccess)
 
-      return unless json?(response)
+      return if response.instance_of?(Net::HTTPNoContent)
```

Comment:
I'm curious why you made this change. Does the `204 No Content` response have a `Content-Type: application/json` header?

Reply:
Ah, nevermind. I just saw your note about `application/octet-stream`.